lib/capistrano/tasks/yarn.rake

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 desc 'Install Javascript dependencies via `yarn`'
-task :yarn_install do
+task yarn_install: :environment do
```

Comment:
This should be reverted.  Loading the environment is slow and is not useful in this case.  Rubocop was overreaching.

Reply:
done.